### ISSUE TITLE
jackson-databind-2.5.3.jar How to fix too low version

### ISSUE THREAD

**Issue by slcnx** [jenkinsci/git-changelog-plugin#80]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Paste the output here
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

centos 7

### Reproduction steps

官方报告的

### Expected Results

升级版本，或编译changelog-plugin，修复bug

### Actual Results

尚未修复

**Comment by tomasbjerre** [comment]
What is the problem?

**Comment by tomasbjerre** [comment]
I'm guessing you want a newer version of Jackson. It is a transitive dependency:

```
runtimeClasspath - Runtime classpath of source set 'main'.
+--- com.github.jknack:handlebars:4.3.1
|    \--- org.slf4j:slf4j-api:1.7.32 -> 1.8.0-beta2
+--- com.squareup.retrofit2:retrofit:2.9.0
|    \--- com.squareup.okhttp3:okhttp:3.14.9
|         \--- com.squareup.okio:okio:1.17.2
+--- com.squareup.retrofit2:converter-gson:2.9.0
|    +--- com.squareup.retrofit2:retrofit:2.9.0 (*)
|    \--- com.google.code.gson:gson:2.8.5
+--- com.jayway.jsonpath:json-path:2.7.0
|    +--- net.minidev:json-smart:2.4.7
|    |    \--- net.minidev:accessors-smart:2.4.7
|    |         \--- org.ow2.asm:asm:9.1
|    \--- org.slf4j:slf4j-api:1.7.33 -> 1.8.0-beta2
+--- org.eclipse.jgit:org.eclipse.jgit:5.13.1.202206130422-r
|    +--- com.googlecode.javaewah:JavaEWAH:1.1.13
|    \--- org.slf4j:slf4j-api:1.7.30 -> 1.8.0-beta2
\--- org.gitlab:java-gitlab-api:4.1.1
     +--- com.fasterxml.jackson.core:jackson-core:2.5.3
     +--- com.fasterxml.jackson.core:jackson-databind:2.5.3
     |    +--- com.fasterxml.jackson.core:jackson-annotations:2.5.0
     |    \--- com.fasterxml.jackson.core:jackson-core:2.5.3
     +--- commons-io:commons-io:2.4
     +--- org.slf4j:slf4j-api:1.8.0-beta2
     \--- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.5.1
          +--- com.fasterxml.jackson.core:jackson-core:2.5.1 -> 2.5.3
          \--- com.fasterxml.jackson.core:jackson-databind:2.5.1 -> 2.5.3 (*)
```

If there is an issue with Jackson that effects `java-gitlab-api`, you should report it on that library. There is actually already an issue: https://github.com/timols/java-gitlab-api/issues/401

**Comment by slcnx** [comment]
谢谢您的回复，请问一下,如果我更新了org.gitlab:java-gitlab-api:4.1.1这个版本到一个较新的版本，这个git-changelog-plugin的版本如何更新呢？

**Comment by slcnx** [comment]
我尝试了使用 run.sh脚本，但是构建不成功。

**Comment by tomasbjerre** [comment]
Should be fixed now in 3.32
